cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForTransformers.java

```diff
@@ -0,0 +1,391 @@
+package com.powsybl.cgmes.conversion;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.powsybl.cgmes.conversion.RegulatingControlMapping.RegulatingControl;
+import com.powsybl.cgmes.model.CgmesModelException;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.PhaseTapChanger;
+import com.powsybl.iidm.network.PhaseTapChangerAdder;
+import com.powsybl.iidm.network.RatioTapChanger;
+import com.powsybl.iidm.network.RatioTapChangerAdder;
+import com.powsybl.iidm.network.Terminal;
+import com.powsybl.iidm.network.TwoWindingsTransformer;
+import com.powsybl.triplestore.api.PropertyBag;
+
+/**
+ * @author José Antonio Marqués <marquesja at aia.es>
+ * @author Marcos de Miguel <demiguelm at aia.es>
+ */
+
+public class RegulatingControlMappingForTransformers {
+
+    private static final String TAP_CHANGER_CONTROL_ENABLED = "tapChangerControlEnabled";
+
+    public RegulatingControlMappingForTransformers(RegulatingControlMapping parent) {
+        this.parent = parent;
+        this.context = parent.context();
+        t2xMapping = new HashMap<>();
+    }
+
+    public void add(String transformerId, RegulatingControlRatio rcRtc, RegulatingControlPhase rcPtc) {
+        if (t2xMapping.containsKey(transformerId)) {
+            throw new CgmesModelException("Transformer already added, Transformer id : " + transformerId);
+        }
+
+        RegulatingControlForTwoWindingsTransformer rc = new RegulatingControlForTwoWindingsTransformer();
+        rc.ratioTapChanger = rcRtc;
+        rc.phaseTapChanger = rcPtc;
+        t2xMapping.put(transformerId, rc);
+    }
+
+    public RegulatingControlRatio buildEmptyRegulatingControlRatio() {
```

Comment:
Is this method really useful? (see below)

Reply:
Deleted